### ISSUE TITLE
host model ile çalışmak isteyen firmalar için merkezi mysql yapısı anlatılacak

### ISSUE THREAD

**Issue by mdpinar** [seyhanp/seyhan-docs#1]

**Comment by khaluk** [comment]
@mdpinar bu issue ile ilgili biraz daha ayrıntılı bilgi verebilir misiniz?
Hangi konuları içermeli?

**Comment by mdpinar** [comment]
evet, elbette. kullanıcılar zaman zaman php hosting sistemi ile çalışan sunucularında seyhan'ı çalıştırmak istediklerin ve bunun için ne yapabileceklerini soruyorlar. php hostinglerde java jdk kurulamadığı için maalesef bu mümkün olmuyor. bunun yerine, hosting üzerinde bulunan veritabanını uzaktan ortak olarak kullanma seçeneği cazip oluyor.

bu durumda, kullanıcılar isterlerse, hosting sunucuları üzerindeki veritabanını kullanarak ve seyhan'ı da istedikleri yere kurarak (db ayarlarını yapmak kaydı ile) istedikleri gibi uzaktan ve tek db yapısında kullanabilirler.

bu soru zaman zaman geldiği için, ben de bunu anlatan bir döküman yazayım demiştim.
